src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -98,6 +98,8 @@ public class SnowflakeSinkConnectorConfig {
   // Snowflake Metadata Flags
   private static final String SNOWFLAKE_METADATA_FLAGS = "Snowflake Metadata Flags";
   public static final String SNOWFLAKE_METADATA_CREATETIME = "snowflake.metadata.createtime";
+  public static final String SNOWFLAKE_METADATA_CONNECTOR_PUSH_TIME =
+      "snowflake.metadata.connectorPushTime";
```

Comment:
this field is applicable to streaming only I suppose, good to add comment. 
adding streaming specific name would mean we wont be able to add for snowpipe in future. SNowpipe wont need it since they have commit time in load history but just FYI.

Reply:
Agree, I think it's always good to be more clear in the property name even though more detail will be in the comment, I suggest we call it `snowflake.streaming.metadata.approximate.flushtime` (or pushtime). I would not concern about it being too long since it won't see part of the every row, plus we have pretty long ones already like `"value.converter.basic.auth.credentials.source"`

Reply:
Adding `streaming` in the property name makes perfect sense. I believe `snowflake.streaming.metadata.connectorPushTime` is quite descriptive.